odoo/fields.py

```diff
@@ -939,6 +939,10 @@ def update_db(self, model, columns):
                 # discard the "classical" computation
                 return False
 
+        if self.compute and skip_recomputation:
```

Comment:
Is `self.compute` necessary ?

Reply:
It would work without it too

https://github.com/odoo/odoo/blob/04c58aa229781ad165492765414483ae8e8d6693/odoo/models.py#L2603-L2605

My idea was to change return value only when it's really needed